pom.xml

```diff
@@ -59,6 +59,15 @@
                             <file>${project.basedir}/resources/CGMES/CGMES_v2.4.15_MicroGridTestConfiguration_BD_v2/MicroGridTestConfiguration_EQ_BD.xml</file>
                             <file>${project.basedir}/resources/CGMES/CGMES_v2.4.15_MicroGridTestConfiguration_BD_v2/MicroGridTestConfiguration_TP_BD.xml</file>
                             <file>${project.basedir}/resources/IEEE/ieee300cdf.txt</file>
+                            <file>${project.basedir}/resources/MATPOWER/ieee9.json</file>
```

Comment:
I would not add these files as it is not really matpower files but ref file for tests

Reply:
I agree: maybe we can convert few files from https://github.com/MATPOWER/matpower/blob/master/data/case6515rte.m (like RTE file)?

Reply:
Why not but this is probably not necessary to provide test cases for any of our importer.